marklogic-data-hub/src/test/ml-modules/root/test/suites/com/marklogic/hub/mapping/mapping-test.sjs

```diff
@@ -14,9 +14,16 @@ function mapsJSONasExpected() {
     },
     "targetEntity" : "Customer"
   }).value.envelope.instance.Customer;
+  const message = `Unexpected output: ${describe(mappedInstance)}`;
   return [
-    test.assertEqual("Bob", mappedInstance.firstname, `Unexpected output: ${describe(mappedInstance)}`),
-    test.assertEqual("Customer-1234", mappedInstance.id, `Unexpected output: ${describe(mappedInstance)}`)
+    test.assertEqual("Bob", mappedInstance.firstname, message),
+    test.assertEqual("Customer-1234", mappedInstance.id, message),
+    test.assertEqual(10, mappedInstance.firstNumber, message),
+    test.assertEqual(null, mappedInstance.secondNumber, message),
+    test.assertEqual(10, mappedInstance.thirdNumber, message),
+    test.assertEqual(10, mappedInstance.fourthNumber, message),
+    test.assertEqual(null, mappedInstance.fifthNumber, message),
+    test.assertEqual(null, mappedInstance.sixthNumber, message)
```

Comment:
This test makes use of "/data-hub/5/builtins/steps/mapping/default/main.sjs" which is the main.sjs of the old mapping implementation (the one in 5.0). Shouldn't the tests exercise "/data-hub/5/builtins/steps/mapping/entity-services/main.sjs" which is in PR  https://github.com/marklogic/marklogic-data-hub/pull/2839 ?

Reply:
Thanks, I hadn't realized there was a new module. I'll change this test over to use it and see what happens. 

@srinathgit  @ryanjdew  Should "/data-hub/5/builtins/steps/mapping/default/main.sjs" have some kind of deprecation warning in it then? If so, I can add that.